Dokan/Dokan/Scenes/ProductDetails/Cell/ReviewerTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ReviewerTableViewCell.swift
+//  Dokan
+//
+//  Created by raniazeid on 28/07/2022.
+//
+
+import UIKit
+
+class ReviewerTableViewCell: UITableViewCell {
+
+   @IBOutlet weak var reviewerImageView: UIImageView!
+    
+    @IBOutlet weak var reviewerNameLabel: UILabel!
+    
+    @IBOutlet weak var reviewerComment: UITextView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    //MARK: - method to configure cell
+    func configureCell(reviewTableViewCellViewModel:ReviewTableViewCellViewModel){
+        reviewerNameLabel.text = reviewTableViewCellViewModel.reviewerName
+        reviewerComment.text = reviewTableViewCellViewModel.reviewerComment
+        
```

Comment:
Where's the image configuration?

Reply:
waiting king fisher approve because the image configuration url..

Reply:
let's add #warning("SOME MESSAGE")